packages/web-components/src/components/button-group/button-group.ts

```diff
@@ -22,7 +23,9 @@ const { stablePrefix: ddsPrefix } = settings;
  * @element dds-button-group
  */
 @customElement(`${ddsPrefix}-button-group`)
-class DDSButtonGroup extends StableSelectorMixin(LitElement) {
+class DDSButtonGroup extends StableSelectorMixin(CarbonBase) {
+  carbonStyles = ['resetStyles', 'typeStyles'];
```

Comment:
If every component uses `resetStyles` and `typeStyles`, would it make sense to omit them here and always send them in the `carbonStyle` utility anyway?